source/braille.py

```diff
@@ -1398,18 +1406,23 @@ def getFocusContextRegions(obj, oldFocusRegions=None):
 			# No common regions were found.
 			commonRegionsEnd = 0
 			newAncestorsStart = 1
-		# Yield the common regions.
+		# Yield the common regions, thereby setting focusToHardLeft back to False.

```

Comment:
Hmm, no, the current implementation still relies on setting focusTohardLeft to False. Let me investigate this in more detail and come up with a descriptive comment. Cheers for unit testing.